keyboards/zeal60/zeal60_keycodes.h

```diff
@@ -41,6 +41,25 @@ enum zeal60_keycodes {
 	MACRO15,
 };
 
+enum user_keycodes {
```

Comment:
@Wilba6582 Since this touches your keyboards...  Was this okay, or not?

Reply:
Yes, we're putting them here to avoid another #include and propagate this to all keyboards using copypasta VIA support, prior to refactoring this to a generalized solution.

Reply:
Awesome!

I wanted to make sure before doing anything with it.